### PR TITLE
style(frontend): add crypto logos to the header when signed out

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We want to display some specific token icons when the user is logged out.

### Large screens
Screenshot 2024-07-22 at 11 58 42

### Small screens
Screenshot 2024-07-22 at 11 58 50

### CODE REVIEW

File: src/frontend/src/lib/components/hero/HeroSignIn.svelte [hunk 1]
```diff
@@ -2,9 +2,20 @@
...
+	import IconHeaderETH from '$lib/components/icons/IconHeaderETH.svelte';
+	import IconHeaderUSDC from '$lib/components/icons/IconHeaderUSDC.svelte';
 </script>
 
-<div class="mt-6 xl:mt-12 mb-10 pt-2">
+<div class="flex -space-x-2.5 mt-6 xl:mt-12">
+	<div class="z-0"><IconHeaderICP /></div>
+	<div class="z-10"><IconHeaderBTC /></div>
+	<div class="z-20"><IconHeaderETH /></div>
+	<div class="z-30"><IconHeaderUSDC /></div>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/hero/HeroSignIn.svelte:15]: Just to be sure, there is no z-index conflicts with the backdrop of the busy spinner that is displayed when user sign-in right?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/HeroSignIn.svelte:15]: you mean this one, right?

Screenshot 2024-07-22 at 12 52 53

Comment (peterpeterparker) [reply src/frontend/src/lib/components/hero/HeroSignIn.svelte:15]: Yes. Good!

Never too sure with z-indexes 😅

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

File: src/frontend/src/lib/components/hero/HeroSignIn.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/HeroSignIn.svelte:15]: you were right! it gave issues in the end! see PR #1818

**Pull request merged**